src/points/PointsHome.tsx

```diff
@@ -121,7 +122,12 @@ export default function PointsHome({ route, navigation }: Props) {
               />
             </View>
             <View style={styles.balanceRow}>
-              <Text style={styles.balance}>{pointsBalance}</Text>
+              <NumberTicker
+                testID="PointsBalance"
+                textStyles={styles.balance}
+                textHeight={48}
+                finalValue={pointsBalance.toString()}
```

Comment:
i wonder if this prop could be simpler?
* named just `value` 
* of a number type

Reply:
i used a string in case we want to pass in a very big number in here or a localised number (e.g. 12,34) - i'm not sure if these scenarios are likely to be used, but i need to understand the number of digits to animate and the easiest way to get that is to turn it into a string anyway. i guess i don't see many downsides of using a string?

Reply:
yes, this makes totally sense